Tests/Rules/AvoidAssignmentToAutomaticVariable.tests.ps1

```diff
@@ -63,6 +63,21 @@ Describe "AvoidAssignmentToAutomaticVariables" {
             $warnings.Count | Should -Be 0
         }
 
+        It "Does not throw a NullReferenceException when using assigning a .Net property to a .Net property (Bug in 1.17.0 - issue 1007)" {
+            $exceptionThrown = $false
+            try
```

Comment:
shouldn't this just be:
`{ Invoke-ScriptAnalyzer -ScriptDefinition '[foo]::bar = [baz]::quz' -ErrorAction Stop } | Should Not Throw`
It will make the log easier to understand if it fails.

Reply:
Pester only works with terminating errors. PSSA throws non-terminating errors, see https://github.com/pester/Pester/issues/366

Reply:
i would have thought that `-ErrorAction Stop` would do the trick (after all, it's in a try/catch, so you're doing the same thing) so somebody is halting the pipeline